filters/transformFromQueryParams.go

```diff
@@ -30,6 +31,19 @@ func (t *transformFromQueryParams) CreateOptions(ctx *ImageFilterContext) (*bimg
 	// Get crop prams from the request
 	params, ok := ctx.Parameters[cropParameters]
 	if !ok {
+
+		params, ok = ctx.Parameters[focalPointCropParameters]
+
+		if !ok {
+			return &bimg.Options{}, nil
+		}
+
+		params = strings.Split(params[0], ",")
+		if len(params) != 5 {
+			return &bimg.Options{}, nil
+		}
+
+		// TODO do focal point crop
```

Comment:
@duergner can you please open a ticket for this part, so that we don't forget about it?

Reply:
@danpersa done: https://github.com/zalando-stups/skrop/issues/99